src/library.js

```diff
@@ -3267,7 +3267,15 @@ addToLibrary({
     throw 'unwind';
   },
 
-  emscripten_force_exit__deps: ['exit',
+  _emscripten_runtime_keepalive_clear__proxy: 'sync',
```

Comment:
runtime_keepalive_push and runtime_keepalive_pop are thread local.. so its little confusing that this would be proxied.

Reply:
I guess the name is not perfect but since its an internal-only name I doesn't matter too much.